package.json

```diff
@@ -44,7 +44,7 @@
     "regenerator-runtime": "^0.13.7",
     "text-encoding": "^0.7.0",
     "typescript": "^4.3.5",
-    "vite": "^2.6.14",
+    "vite": "todo: wait for next Vite release",
```

Comment:
It looked like this may have been solved via https://github.com/vitejs/vite/issues/6725 -> https://github.com/vitejs/vite/pull/7073 (literally 2 days ago) and not in a Vite release yet but it only seems to fix the problem for `*.js?url` or `*.json?url`.

I played around with that fix and it still doesn't fix the situation for `*.html?url` or non-ESM `.js?url` imports, see https://github.com/vitejs/vite/pull/7097

Reply:
Another Vite community member created a fix after seeing my tests, https://github.com/vitejs/vite/pull/7098, which is now merged and in the latest Vite release. Need to pull it in and see if it works for sure 🙂

Reply:
Seems to work ✅